### DISCUSSION TITLE
Messaging Inbox feature

### DISCUSSION THREAD

**Discussion by Jassi10000-zz** [community/community#3182] [Discussions]
From my experience of using Github , I have experienced that while we follow someone , or while contributing to an open source project , many times I personally want to talk to that person , and also I search  him on Linkedln for personal chat , but in most cases its very tiring as the person can be available with different social names
So its my request or maybe a recommendation that Github should have messaging inbox feature , I think after this feature , the number of open source contributors will also increase when they will ask their doubts or do some personal chat with someone at that point only.

**Comment by evi-liu** [comment]
Thanks for the feedback! We are using this thread to prioritize by gauging customer impact via upvotes and reactions.

**Reply by Jassi10000-zz** [reply]
Ty mam for consideration , Hoping for a nice feedback from GITHUBers

**Comment by PranamBhat** [comment]
The best way to communicate in GitHub is - Go to particular user profile. On the left side, you will see their email addess. Just send a query/help depending on the need. Since GitHub is more like a professional platform, we can't expect chat/call window on this platform. Then it will be more into Social Media and less into Coding!!!

I have a code for getting email address of a particular **GitHub** user. Just execute this code:

```
<input id=username type="text" placeholder="github username or repo link">
<button onclick="fetch(`https://api.github.com/users/${username.value.replace(/^.*com[/]([^/]*).*$/,'$1')}/events/public`).then(e=> e.json()).then(e => [...new Set([].concat.apply([],e.filter(x => x.type==='PushEvent').map(x => x.payload.commits.map(c => c.author.email)))).values()]).then(x => results.innerText = x)">GO</button>
<div id=results></div>

```

Also, I think for more help, you can put your query under **GitHub** "feedback" section.

**Reply by Jassi10000-zz** [reply]
This uses RegEx objects , its gr8 🤝, will try that 
Thanks @PranamBhat 

Anyways , my suggestion was not like to make it a social media , but it's rather easy , coz being a user , I have emailed some users , very very less of them reply to that , that's why to make communication possible I was suggesting this feature
